reposettings.yml

```diff
@@ -1,133 +1,130 @@
-# Coreint settings
-
-labels-common: &labels-common
-  bug:
-    color: e11d21
-    description: Categorizes issue or PR as related to a bug.
-    replaces: [ kind/bug ]
-  cleanup:
-    color: c7def8
-    description: Categorizes issue or PR as related to cleaning up code, process, or technical debt.
-    replaces: [ kind/cleanup ]
-  sustain:
-    color: c7def8
-    description: Categorizes issue or PR as related to sustainability.
-    replaces: [ kind/sustain ]
-  deprecation:
-    color: e11d21
-    description: Categorizes issue or PR as related to a feature/enhancement marked for deprecation.
-    replaces: [ kind/deprecation ]
-  design:
-    color: c7def8
-    description: Categorizes issue or PR as related to design.
-    replaces: [ kind/design ]
-  documentation:
-    color: 0075ca
-    description: Categorizes issue or PR as related to documentation.
-    replaces: [ kind/documentation, doc ]
-  test:
-    color: e11d21
-    description: Categorizes issue or PR as related to a consistently or frequently failing test.
-    replaces: [ kind/test ]
-  feature request:
-    color: c7def8
-    description: Categorizes issue or PR as related to a new feature or enhancement.
-    replaces: [ kind/feature, enhancement, feature ]
-  regression:
-    color: e11d21
-    description: Categorizes issue or PR as related to a regression from a prior release.
-    replaces: [ kind/regression ]
-  support:
-    color: d876e3
-    description: Categorizes issue or PR as a support question.
-    replaces: [ kind/support, question ]
+coreint-settings: &coreint-settings
```

Comment:
I now it is already in the readme, but since we are leaving the unused config file here. Should we also add a comment for us to remember that this is not used anymore?

Reply:
Fair: [#b18cada](https://github.com/newrelic/coreint-automation/pull/79/commits/b18cada883beeb4ba0609946af64b3f2802a0506)